src/App.js

```diff
@@ -13,20 +13,28 @@ const routes = {
     landing: '/'
 };
 
+export const PermissionContext = createContext();
```

Comment:
Since we can use hooks you should check https://reactjs.org/docs/hooks-reference.html#usecontext

Reply:
I updated body.js to use the hook version instead of the `Consumer`, if that's not what you meant, I can update something else!

Reply:
Naah, that's exactly what I wanted.